### ISSUE TITLE
Use higher version on Nodejs in linter action

### ISSUE THREAD

**Issue by ahangarha** [microverseinc/linters-config#196]
In linter files, we have `node-version: "12.x"`. Why version 12?

Isn't it better to use at least the LTS version (16.15.1)?

Having the old version in the linter workflow is making issues for several students. Their `package-lock.json` is build with more recent version of node and npm and the old version cannot handle some package installation.

I propose the upgrade from v12 to LTS version.

**Comment by ahangarha** [comment]
More update:

An example of the error message we see in github action for linter:
![image](https://user-images.githubusercontent.com/11241315/172912242-ffe3b324-e81c-445f-871a-364a867e36bb.png)

I did one thing: I picked up one of my own projects in module 2. That time I didn't have any issue with linters. I removed `node_modules` and `package-lock.json` . Then I ran `npm install`. So `package.json` and `linter.yml` are exact the same. Just `pakage-lock.json` got changed. After that, linter workflow doesn't pass with exact same error.

This means in last 2-4 months, some packages got upgraded in such a way that Node 12 and relevant npm version cannot resolve dependencies and install them anymore.

So as per my understanding, we should do either of these options:
- Explicitly mention the exact version of packages to be installed (not the minimum version)
- Upgrage Nodejs and npm versions.
